docs/samples-library.md

```diff
@@ -0,0 +1,45 @@
+---
+id: samples-library
+title: Application samples library
+sidebar_label: Samples library
+---
+
+- [**Polyglot application**](https://github.com/temporalio/temporal-polyglot): Showcases how Workflow Executions, that are written in different languages, can send messages to each other.
+  Go, Java, PHP, and NodeJS SDKs are represented in this sample.
+  It also shows how to properly propagate errors including how to do it across Workflows written in different languages.
+
+## Go
+
+- [**Hello world application template in Go**](https://github.com/temporalio/hello-world-project-template-go): Provides a quick start development app for users.
+  Works in conjunction with the ["Hello World!" from scratch tutorial in Java](/docs/go/hello-world-tutorial).
+
+- [**Money transfer application template in Go**](https://github.com/temporalio/money-transfer-project-template-go): Provides a quick start development app for users.
+  It demonstrates a basic money transfer Workflow Definition and works in conjunction with the [Run your first app tutorial in Go](/docs/go/run-your-first-app-tutorial).
+
+- [**Subscription style Workflow Definition in Go**](https://github.com/temporalio/subscription-workflow-project-template-go): Demonstrates some of the patterns that could be implemented for a subscription style business process.
+
+- [**eCommerce application example in Go**](https://github.com/temporalio/temporal-ecommerce): Showcases a per user shopping cart style Workflow Definition that includes an API for adding and removing items from the cart as well as a web UI.
+  This application sample works in conjunction with the [eCommerce in Go tutorial](/blog/tags/go-ecommerce-tutorial).
+
```

Comment:
Perhaps add a para such as the following (either here or in the source)?

The following sections describe the available Go samples and how to run them.

Reply:
Going to revise those READMEs so that the format is the same.